Horizon/Horizon/Sources/Features/Dashboard/View/DashboardViewModel.swift

```diff
@@ -33,14 +33,31 @@ final class DashboardViewModel: ObservableObject {
 
     // MARK: - Init
 
-    init(interactor: GetProgramsInteractor) {
+    init(
+        programsInteractor: GetProgramsInteractor,
+        userInteractor: GetUserInteractor
+    ) {
         unowned let unownedSelf = self
 
-        interactor.getPrograms()
+        programsInteractor.getPrograms()
             .sink { programs in
                 unownedSelf.programs = programs
                 unownedSelf.state = .data
             }
             .store(in: &subscriptions)
+
+        userInteractor.getUser()
+            .map { $0.name }
+            .map { "Hi, \($0)" }
+            .replaceError(with: "")
+            .assign(to: &$title)
     }
+
+    // MARK: - Inputs
+
```

Comment:
Please rename it to // MARK: - Functions, this is not input

Reply:
They are user inputs.